mono_repo/lib/src/github_config.dart

```diff
@@ -19,7 +22,59 @@ class GitHubConfig {
     Map<String, dynamic> on,
     String cron,
     this.workflows,
-  ) : on = _parseOn(on, cron);
+  ) : on = _parseOn(on, cron) {
+    if (workflows != null) {
+      //
+      // Validate that no workflow uses the reserved key
```

Comment:
This method is 52 lines long with 3 named sections. Consider extracting each into a named method.

Reply:
ack